examples/tutorials/line-plot.py

```diff
@@ -0,0 +1,127 @@
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 8],
+    y=[5, 9],
+    pen="1p,black",
+)
+fig.show()
+
+########################################################################################
+# Additional line segments can be added by including additional values for ``x``
+# and ``y``.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 6, 9],
+    y=[5, 7, 4],
+    pen="1p,black",
+)
+fig.show()
+
+########################################################################################
+# To plot multiple lines, :meth:`pygmt.Figure.plot` needs to be used for each
+# additional line. Parameters such as ``region``, ``projection``, and ``frame`` do
+# not need to be repeated in subsequent uses.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 6, 9],
+    y=[5, 7, 4],
+    pen="2p,blue",
+)
+fig.plot(x=[2, 4, 10], y=[3, 8, 9], pen="2p,red")
+fig.show()
+
+########################################################################################
+# Change line attributes
+# ----------------------
+#
+# The line attributes can be set using the ``pen`` parameter. ``pen`` takes a string
+# argument with the optional values *line width*,*line color*,*line style*.
+#
+# In the example below, the pen width is set to ``"5p"``, and with *black* as the
+# default color and *solid* as the default style.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 8],
+    y=[3, 9],
+    pen="5p",
+)
+fig.show()
+
+########################################################################################
+# The line color can be set and is added after the line width to the ``pen`` argument.
+# In the example below, the line color is set to "red".
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 8],
+    y=[3, 9],
+    pen="5p,red",
+)
+fig.show()
+
+########################################################################################
+# The line style can be set and is added after the line width or color to the
+# ``pen`` argument.  In the example below, the line color is set to *dot dot dash*, and
+# the default color *black* is used.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 8],
+    y=[3, 9],
+    pen="5p,..-",
+)
+fig.show()
+
+########################################################################################
+# The line width, color, and style can all be set in the same ``pen`` argument. In the
+# example below, the line width is set to *7p*, the color is set to *green*, and the
+# line style is *dash dot dash*.
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X25c/20c",
+    frame="a",
+    x=[1, 8],
+    y=[3, 9],
+    pen="7p,green,-.-",
+)
+fig.show()
```

Comment:
I'm thinking whether it's better to merge these 4 figures into a single figure, i.e., showing 4 lines with different line properties. It would save much space and also easier for people to compare the different *pen* settings.

We also have a gallery example for [different line styles](https://pygmt-qalqf8vyi.vercel.app/gallery/line/linestyles.html). I think you can add a link to that example.

Reply:
This was a conscious choice on my part, since the gallery already shows the different *pen* settings side by side. My thought process is that the tutorial instead takes the user through the steps of plotting a line, and shows how they can set one or more properties.

I'm happy to change it if you think its the smarter move, but I do like the way it's currently displayed.

Also, is there a good way to link to other PyGMT pages similar to using `gmt-docs`?

Reply:
OK. let's keep it this way.

Reply:
> Also, is there a good way to link to other PyGMT pages similar to using gmt-docs?

You can use to link to that file:
```
:doc:`/gallery/line/linestyles`
```

Reply:
Thanks. I wasn't having any luck with trying variations of `pygmt-docs`

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html 

You may read this if you want to know more about cross-references in ReST language.